src/couch_jwt_auth.erl

```diff
@@ -58,6 +58,12 @@ decode(Token, Config) ->
       end, List), posix_time(calendar:universal_time()), Config)
   end.
 
+ensure_safe_token(Token, Config) ->
+  case couch_util:get_value(Token, Config) of
```

Comment:
Well, for 2.0 better try to compile project and fix all the ongoing errors. For instance, `couch_config` turned into `config`. I'm not familiar with this codebase to say what else you'll have to do.